src/dash-table/syntax-tree/lexeme/expression.ts

```diff
@@ -0,0 +1,59 @@
+import isNumeric from 'fast-isnumeric';
+import * as R from 'ramda';
+
+import { LexemeType, IUnboundedLexeme } from 'core/syntax-tree/lexicon';
+import { ISyntaxTree } from 'core/syntax-tree/syntaxer';
+import operand from './operand';
+
+const STRING_REGEX = /^(('([^'\\]|\\'|\\)+')|("([^"\\]|\\"|\\)+")|(`([^`\\]|\\`|\\)+`))/;
+const VALUE_REGEX = /^(([^\s'"`{}()\\]|\\[\s'"`{}()]?)+)(?:[\s)]|$)/;
```

Comment:
Similarly here, I'm having a harder time figuring out the consequences, but do we really want ```\\[\s'"`{}()]?```, which means the match could end with a `\`, or do we *always* include *any* character after `\` by just using `\\.`?

Reply:
As discussed, updating to force escaping `\` everywhere and allow escaping any character in all contexts.